src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -274,16 +280,41 @@ def receive_trial_result(self, parameter_id, parameters, value):
             if value is dict, it should have "default" key.
             value is final metrics of the trial.
         """
+        logger.info("value = %s" %value)
         reward = extract_scalar_reward(value)
         # restore the paramsters contains '_index'
         if parameter_id not in self.total_data:
             raise RuntimeError('Received parameter_id not in total_data.')
         params = self.total_data[parameter_id]
 
-        if self.optimize_mode is OptimizeMode.Maximize:
-            reward = -reward
+        if fake:
+            rval = self.CL_rval
```

Comment:
Seems we don't update the fake point here? just log?

Reply:
We will not use fake value to update fake value base.